src/styles/blocks/calendar.scss

```
@@ -0,0 +1,28 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $day-block-size * $days-in-week + $day-block-gap * ($days-in-week - 1);
+  gap: $day-block-gap;
+  padding: 50px;
+  margin: auto;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    width: $day-block-size;
+    height: $day-block-size;
+    background-color: #eee;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    border: 1px solid #000;
+    align-items: center;
+    justify-content: center;
+    transition: 0.5s;
```

you should specify the properties that will change on transition, and better store transition duration in variable
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```

---

ok